pkg/api/pod/warnings.go

```diff
@@ -168,6 +168,56 @@ func warningsForPodSpecAndMeta(fieldPath *field.Path, podSpec *api.PodSpec, meta
 		}
 	}
 
+	items := sets.NewString()
+	for _, v := range podSpec.Volumes {
+		if v.Projected != nil {
+			for _, k := range v.Projected.Sources {
+				if k.ServiceAccountToken != nil {
+					items.Insert(k.ServiceAccountToken.Path)
+				}
+			}
+		}
+
+	}
+
+	if items.Len() > 0 {
+		for i, item := range podSpec.Volumes {
+			if item.Projected != nil {
+				for _, k := range item.Projected.Sources {
+					if k.ConfigMap != nil {
+						for j, item := range k.ConfigMap.Items {
+							if items.Has(item.Path) {
+								warnings = append(warnings, fmt.Sprintf("%s: has duplicated path with ServiceAccountToken %q", fieldPath.Child("spec", "volumes").Index(i).Child("projected", "sources", "configMap").Index(j), item.Path))
```

Comment:
Shouldn't this accumulate all paths and just complain if ANY of them overlap?

Reply:
We already have something similar on the validation step. I believe the problem is that while validating we don't check if the service account token path is unique.

Reply:
Correct me if I'm wrong, but my understanding of the issue here is that we don't check for the uniqueness of the paths in the projected volume service account token with other items from projected volumes. So I believe the issue is scoped to the projected volumes only. That's why I'm checking only projected volumes. Let me know if I'm missing something.